package.json

```diff
@@ -106,7 +126,7 @@
         "visible": true,
         "subPermissions": [
           "ui-organizations.basic.view",
-          "settings.organizations.enabled"
+          "ui-organizations.categories.settings.view"
```

Comment:
IMO it's not the right approach to have organizations view permissions that include settings. For me `Settings` it's a separate permission set

Reply:
updated